packages/lesswrong/server/repos/DatabaseMetadataRepo.ts

```diff
@@ -78,6 +81,56 @@ export default class DatabaseMetadataRepo extends AbstractRepo<"DatabaseMetadata
     return this.getGivingSeasonHearts(electionName);
   }
 
+  async getBannerEmojis(): Promise<BannerEmoji[]> {
+    const result = await this.getRawDb().oneOrNone(`
+      -- DatabaseMetadataRepo.getBannerEmojis
+      SELECT ARRAY_AGG(q."value" || JSONB_BUILD_OBJECT(
+        'userId', u."_id",
+        'displayName', u."displayName"
+      )) "emojis"
+      FROM (
+        SELECT (JSONB_EACH("value")).*
+        FROM "DatabaseMetadata"
+        WHERE "name" = $1
+      ) q
+      JOIN "Users" u ON q."key" = u."_id"
+    `, [BANNER_EMOJI_NAME]);
+    return result?.emojis ?? [];
+  }
+
+  async addBannerEmoji(
+    userId: string,
+    emoji: string,
+    link: string,
+    description: string,
+    x: number,
+    y: number,
+    theta: number,
+  ): Promise<BannerEmoji[]> {
+    const t = new Date().toISOString();
```

Comment:
The time isn't currently used anywhere, but somebody mentioned wanting the times that entries were created when we did the interactive banner back in December so I added it here just in case.